examples/seismic/model.py

```diff
@@ -98,6 +150,89 @@ def demo_model(preset, **kwargs):
         return Model(vp=v, origin=origin, shape=v.shape,
                      spacing=spacing, nbpml=20)
 
+    elif preset.lower() in ['marmousitti', 'marmousi2dtti']:
+
+        dimensions = (201, 201, 70)
+        shape = (201, 70)
+        spacing = (10., 10.)
+        origin = (0., 0.)
+        nbpml = kwargs.pop('nbpml', 20)
+
+        # Read 2D Marmousi model from opesc/data repo
+        data_path = kwargs.pop('data_path', None)
+        if data_path is None:
+            error("Path to opesci/data not found! Please specify with "
+                  "'data_path=<path/to/opesci/data>'")
+            raise ValueError("Path to model data unspecified")
+        path = os.path.join(data_path, 'marmousi3D/vp_marmousi_bi')
+
+        # velocity
+        vp = 1e-3 * np.fromfile(os.path.join(data_path, 'marmousi3D/MarmousiVP.raw'),
+                                dtype='float32', sep="")
+        vp = vp.reshape(dimensions)
+        vp = vp[101, :, :]
+        # Epsilon, in % in file, resale between 0 and 1
+        epsilon = np.fromfile(os.path.join(data_path, 'marmousi3D/MarmousiEps.raw'),
+                              dtype='float32', sep="") * 1e-2
+        epsilon = epsilon.reshape(dimensions)
+        epsilon = epsilon[101, :, :]
+        # Delta, in % in file, resale between 0 and 1
+        delta = np.fromfile(os.path.join(data_path, 'marmousi3D/MarmousiDelta.raw'),
+                            dtype='float32', sep="") * 1e-2
+        delta = delta.reshape(dimensions)
+        delta = delta[101, :, :]
+        # Theta, in degrees in file, resale in radian
+        theta = np.fromfile(os.path.join(data_path, 'marmousi3D/MarmousiTilt.raw'),
+                            dtype='float32', sep="")
+        theta = np.float32(np.pi / 180 * theta.reshape(dimensions))
+        theta = theta[101, :, :]
+
+        return Model(vp=vp, origin=origin, shape=shape,
+                     spacing=spacing, nbpml=nbpml,
+                     epsilon=epsilon, delta=delta, theta=theta,
+                     **kwargs)
+
+    elif preset.lower() in ['marmousi-tti3d', 'marmousi3d-tti']:
+
+        dimensions = (201, 201, 70)
+        spacing = (10., 10., 10.)
+        origin = (0., 0., 0.)
+        nbpml = kwargs.pop('nbpml', 20)
+
+        # Read 2D Marmousi model from opesc/data repo
+        data_path = kwargs.pop('data_path', None)
+        if data_path is None:
+            error("Path to opesci/data not found! Please specify with "
+                  "'data_path=<path/to/opesci/data>'")
+            raise ValueError("Path to model data unspecified")
+        path = os.path.join(data_path, 'marmousi3D/vp_marmousi_bi')
+
+        # Velcoity
+        vp = 1e-3 * np.fromfile(os.path.join(data_path, 'marmousi3D/MarmousiVP.raw'),
```

Comment:
How does one get these ``.raw`` files ? Should we add an optional requirement to Devito to run these examples ? ( @mlange05 )

Reply:
https://github.com/opesci/data/tree/master/marmousi3D

Reply:
Can add requirement if needed

Reply:
@mlange05 perhaps we can have it optional ? and update the error message above accordingly?